app/views/dashboards/_actions.html.erb

```diff
@@ -1,35 +1,49 @@
-<nav class="hidden m:block">
-  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) %>" href="/dashboard">
+<nav class="hidden m:block" aria-label="Dashboards">
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) %>"
+    href="<%= dashboard_path %>"
+    <%= params[:action] == "show" && (params[:which] == "organization" || params[:which].blank?) ? ' aria-current="page"'.html_safe : "" %>>
     Posts
     <span class="crayons-indicator"><%= @user.articles_count %></span>
   </a>
 
-  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?) %>" href="/<%= current_user.username %>/series">
+  <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:action] == "series" && (params[:which] == "organization" || params[:which].blank?) %>"
```

Comment:
Probably for another PR as we use `crayons-link--current` in a lot of places, but instead of having a `crayons-link--current` class, that styling could be handled via `aria-current`, e.g.

```css
.crayons-link[aria-current="page"]
```

This would also promote using the `aria-current` attribute.

Reply:
I love this! Yes definitely let's do it, but yeah it can be a separate PR that we can regression test thoroughly